Orange/widgets/visualize/owboxplot.py

```diff
@@ -46,60 +46,27 @@ def compute_scale(min_, max_):
     return first_val, step
 
 
-def _quantiles(a, freq, q, interpolation="midpoint"):
```

Comment:
This was needed to compute quartiles from distributions. Now quartiles are computed from data, using `np.nanquartiles`.